modules/checklist/assets/js/app/components/success-message.js

```diff
@@ -0,0 +1,42 @@
+import { Button, Card, CardActions, CardContent, CardMedia, Typography } from '@elementor/ui';
+
+const SuccessMessage = () => {
+	const hideChecklist = () => {
+		$e.run( 'document/elements/settings', {
+			container: elementor.settings.editorPreferences.getEditedView().getContainer(),
+			settings: {
+				show_launchpad_checklist: '',
+			},
+			options: {
+				external: true,
+			},
+		} );
+	};
```

Comment:
This is too good to be here IMO
I would move it to the utils/functions.js to be used here and later in more places